src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/view/CollectionTabPane.kt

```diff
@@ -0,0 +1,41 @@
+package org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.view
+
+import javafx.collections.ObservableList
+import javafx.event.EventTarget
+import javafx.scene.control.TabPane
+import org.wycliffeassociates.otter.jvm.app.theme.AppStyles
+import tornadofx.*
+
+class CollectionTabPane(private val sourceItems: ObservableList<String>) : TabPane() {
+
+    init {
+        initTabs()
+        sourceItems.onChange {
+            initTabs()
+        }
+    }
+
+    private fun initTabs() {
+        tabs.clear()
+        createTab("bundle")
+        sourceItems.forEach {
+            createTab(it)
+        }
+    }
+
+    private fun createTab(type: String) {
+        val labelGraphicPair = when (type) {
+            "bundle" -> Pair("Scripture", null) // TODO: Enums
+            "help" -> Pair("tN", AppStyles.tNGraphic())
```

Comment:
Need to use something other than type.  tQ will also be of type "help".

Reply:
changed to using identifier